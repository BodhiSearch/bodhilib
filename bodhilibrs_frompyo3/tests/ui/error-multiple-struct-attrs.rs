use frompyo3::FromPyO3;

#[derive(FromPyO3)]
#[frompyo3(name = "foo")]
#[frompyo3(name = "bar")]
struct TestMultipleStructAttrs {}

fn main() {}
