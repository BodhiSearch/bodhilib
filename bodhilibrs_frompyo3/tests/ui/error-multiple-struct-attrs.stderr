error: Multiple attributes not supported
 --> tests/ui/error-multiple-struct-attrs.rs:5:1
  |
5 | #[frompyo3(name = "bar")]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^
